otx/algorithms/action/adapters/openvino/task.py

```diff
@@ -147,20 +141,26 @@ def forward(self, image: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:
         return self.model.infer_sync(image)
 
 
-class DataLoaderWrapper(DataLoader):
+class DataLoaderWrapper:
     """DataLoader implementation for ActionOpenVINOTask."""
 
-    def __init__(self, dataloader: DataLoader, inferencer: BaseInferencer):
-        super().__init__(config=None)
+    def __init__(self, dataloader: Any, inferencer: BaseInferencer, shuffle: bool = True):
```

Comment:
Normally openvino tasks are used to inference. Therefore, I think default value for shuffle should be False.
What do you think?

Reply:
This dataloader is used for PTQ optimize only, when we have to manually apply preprocessing.
By design it shouldn't be applied outside of PTQ optimization, so I set this field to true for compliance with old POT config.